CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 ### Changed
 
 * Added `RSpec/ExampleLength` cop to style guide. ([@a.branzeanu][])
+* Disable `RSpec/NestedGroups` cop by default. ([@r.dubrovsky][])
```

Comment:
`Disable` is imperative. The changelog is about what has been done, not about what should be done, so better to put it in the past: 'DisableD`.

Reply:
@texpert I prefer to leave a present simple here so that our changelog and release notes will be consistent (we used a Present Simple for all changes below). 

Yep, this is a little bit confused that we have titles in past tense but changed items in the present tense but this is just a historical issue. (Firstly I added titles in the past tense (added, removed, changed and etc.)  and then decided to add changelog notes in the present tense. I just investigate releases and changelog of rubocop libraries often and I found that they use the present simple for those notes. Only after some time, I recognized that they also use titles in present tense too. But I just skipped this issue cause I didn't want to change existing notes. 

So if that issue is really confusing you so just add the issue (and we can fix it later)

Reply:
Issue created - https://github.com/datarockets/datarockets-style/issues/123